### ISSUE TITLE
Callback for subscribeToTopic & unsubscribeFromTopic

### ISSUE THREAD

**Issue by skoften** [hansemannn/titanium-firebase-cloud-messaging#117]
Feature request; callback for **subscribeToTopic** and **unsubscribeFromTopic**, so we can check if the subscription is successful (or not). Missing on Android and iOS if I am correct.

**Comment by m1ga** [comment]
Good idea! I'll add that to my Android update (https://firebase.google.com/docs/cloud-messaging/android/topic-messaging#subscribe_the_client_app_to_a_topic) :+1:

**Comment by m1ga** [comment]
https://github.com/hansemannn/titanium-firebase-cloud-messaging/pull/115 - for Android:
two events: `subscribe` and `unsubscribe` with `(un)subscribed:true/false` as a callback parameter
